.env.tpl

```diff
@@ -13,6 +13,8 @@
 
 EIPFS_INDEXER_SQS_ARN = 'arn:aws:sqs:us-west-2:505595374361:staging-ep-indexer-topic'
 EIPFS_INDEXER_SQS_URL = 'https://sqs.us-west-2.amazonaws.com/505595374361/staging-ep-indexer-topic'
+EIPFS_MULTIHASHES_SQS_ARN = 'arn:aws:sqs:us-west-2:505595374361:multihashes-topic'
```

Comment:
There is no "staging-multihashes-topic"?

Reply:
AFAICT no